rusoto/credential_service_mock/src/main.rs

```diff
@@ -0,0 +1,21 @@
+use warp::{self, path, Filter};
+
+fn main() {
+    let instance_profile_role = path!("latest" / "meta-data" / "iam" / "security-credentials")
+        .map(|| "testrole");
+    let instance_profile_creds = path!("latest" / "meta-data" / "iam" / "security-credentials" / "testrole")
+        .map(|| r#"{
+  "Code" : "Success",
```

Comment:
This response is copy/paste from [moe](https://github.com/matthewkmayer/moe), but all in Rust.